packages/manager/modules/overthebox/src/overthebox/autoconfigure/dhcp/template.html

```diff
@@ -0,0 +1,290 @@
+                        ></span>
+                    </oui-button>
+                    <oui-button
+                        class="ml-2 mb-2"
+                        variant="primary"
+                        on-click="$ctrl.cancelCreateConfig()"
+                    >
+                        <span
+                            data-translate="overTheBox_autoconfigure_cancel"
+                        ></span>
+                    </oui-button>
+                </div>
+            </form>
+        </div>
+        <oui-button
+            variant="primary"
+            data-on-click="$ctrl.addConfig()"
+            class="mb-2 d-flex justify-content-center"
+            data-ng-if="$ctrl.displayAddConfigButton"
+        >
+            <span
+                data-translate="overTheBox_autoconfigure_dhcp_config_add"
+            ></span>
+        </oui-button>
+        <oui-datagrid
+            page-size="5"
+            rows="$ctrl.configList"
+            data-ng-if="$ctrl.configList.length > 0"
+        >
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_interface' | translate"
+                property="configuration.interface"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_offset' | translate"
+                property="configuration.offset"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_pool_size' | translate"
+                property="configuration.poolSize"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_lease_duration' | translate"
+                property="configuration.leaseDuration"
+            ></oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_priority' | translate"
+                property="priority"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column>
+                <oui-button
+                    variant="primary"
+                    size="s"
+                    on-click="$ctrl.removeConfig($row)"
+                >
+                    <span
+                        class="oui-icon oui-icon-trash"
+                        aria-hidden="true"
+                    ></span>
+                </oui-button>
+            </oui-datagrid-column>
+        </oui-datagrid>
+    </div>
+    <div class="oui-box oui-box_light mt-2">
+        <h4
+            class="oui-box__heading"
+            data-translate="overTheBox_autoconfigure_dhcp_static_leases"
+        ></h4>
+        <div data-ng-if="$ctrl.staticLeases.length === 0" class="mb-2">
+            <em data-translate="overTheBox_autoconfigure_no_values"></em>
+        </div>
+        <div data-ng-if="$ctrl.showAddStaticLease">
+            <form novalidate name="formStaticLease">
```

Comment:
**nitpick:** Parent `div` is not required. ngIf can be applied directly on the `<form>` element.

Reply:
Can we move the `ngIf` directly on the `<form>` element? (and get rid of the `<div>` element).
Thanks